detection/notebooks/00_webcam.ipynb

```diff
@@ -0,0 +1,650 @@
+{
```

Comment:
<p>Could explain here on a high level what the parameters below do. I.e. "To speed up model inference we restrict processing to use at most an image resolution of 200 pixels, and only use up to 5 candidate object proposals per image. For an information on the underlying technology see the [FAQ](FAQ.md) and the [training introduction notebook](LINK) in this directory</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/ComputerVision/pull/296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>